### PR TITLE
Mention file system watching being enabled by default in release notes

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

**Comment by lptr** [comment]
@bot-gradle test QFL please.

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -27,7 +27,23 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 
 <!-- Do not add breaking changes or deprecations here! Add them to the upgrade guide instead. --> 
 
-## Performance improvements for incremental development
+## Performance improvements
```
Comment (wolfs) [inline subprojects/docs/src/docs/release/notes.md:30]: File system watching is also an improvement for incremental development. How about we keep the this header `Performance improvements for incremental development`, and call the next header `Android performance improvements` or something along those lines. WDYT?

**Review by wolfs — COMMENTED** [review]
Looks good! I left one comment.

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (lptr) [reply subprojects/docs/src/docs/release/notes.md:30]: I'm not sure. Would you mind making your proposal a commit to this PR?

Comment (wolfs) [reply subprojects/docs/src/docs/release/notes.md]: How about now?

Comment (lptr) [reply subprojects/docs/src/docs/release/notes.md]: Let's go for it!

**Comment by lptr** [comment]
@pioterj PTAL. Once you are okay with this we can merge it.

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (pioterj) [reply subprojects/docs/src/docs/release/notes.md]: I'd keep the section name as "performance improvements" for consistency with other release notes.

**Pull request merged**